Dockerfile

```diff
@@ -23,15 +23,23 @@ RUN mkdir -p /config/.config/openbox \
     && echo '<?xml version="1.0" encoding="UTF-8"?> \
 <openbox_config> \
   <applications> \
-    <application  type="normal" class="jetbrains-studio"> \
+    <application type="normal" class="jetbrains-studio"> \
       <decor>no</decor> \
       <fullscreen>yes</fullscreen> \
     </application> \
   </applications> \
 </openbox_config>' > /config/.config/openbox/rc.xml
 
 
-CMD ["/init"]
+# Add security settings for opening links
+RUN apk add --no-cache chromium \
+    && mkdir -p /config/.config/chromium \
+    && echo "user_pref('network.protocol-handler.warn-external-default', true);" > /config/.config/chromium/prefs.js
+
+# Enable nested virtualization
+RUN apk add --no-cache qemu qemu-system-x86_64 \
+    && echo "options kvm_intel nested=1" > /etc/modprobe.d/kvm_intel.conf
```

Comment:
We might need to check first for Intel/AMD, and then define the conf.


Wdyt?

Reply:
I have an AMD host at home I can give this a test on later. Probably also needs to consider what ARM support would look like, but a good start for the prototype.